### PR TITLE
Fix klarna e2e test

### PR DESCRIPTION
**Pull request by leungkinghin**
**Situation**
We have e2e test for Klarna payment

**Complication**
Recently a new IBAN input field is added in Klarna page. It didn't exist before so we don't handle it in our web test, which cause e2e test failure in these days.

**Resolution**
Autofill the IBAN input field in e2e test

**Hints for reviewer**
- Add auto-filling for new IBAN input field.
- Enhance the error logging in klarna payment e2e test
- Change from `page.waitForTimeout(...)` to `frame.waitForSelector(...)` to ensure Klarna payment page is completely loaded

### CODE REVIEW

File: extension/test/e2e/klarna.spec.js [hunk 1]
```diff
@@ -71,11 +71,14 @@ describe('::klarnaPayment::', () => {
...
         const baseUrl = config.getModuleConfig().apiExtensionBaseUrl
         const clientKey = config.getAdyenConfig(adyenMerchantAccount).clientKey
+        const startTime = new Date().getTime()
         const payment = await createPayment(
           ctpClient,
           adyenMerchantAccount,
           ctpProjectKey
         )
+        const endTime = new Date().getTime()
+        logger.debug('klarna::createPayment:', endTime - startTime)
```
Comment (ahmetoz) [inline extension/test/e2e/klarna.spec.js:81]:
```suggestion
        logger.debug('klarna::createPayment:elapsedMilliseconds', endTime - startTime)
```

**Review by ahmetoz — APPROVED** [review]

File: extension/test/e2e/klarna.spec.js (hunk 1 above)
Comment (leungkinghin) [reply extension/test/e2e/klarna.spec.js]: https://github.com/commercetools/commercetools-adyen-integration/pull/831/commits/db5dab508cab646828db5e6306b4cf38b3373817

File: extension/test/e2e/pageObjects/MakePaymentFormPage.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ module.exports = class MakePaymentFormPage {
   async generateAdyenMakePaymentForm(clientKey) {
     await this.page.type('#adyen-client-key', clientKey)
     await this.page.$eval('#adyen-client-key', (e) => e.blur())
-    await this.page.waitForTimeout(10_000)
+    await this.page.waitForTimeout(2_000)
```
Comment (butenkor) [inline extension/test/e2e/pageObjects/MakePaymentFormPage.js:14]: Why this change? I guess there was a reason why we had 10 sec previously?

Comment (ahmetoz) [reply extension/test/e2e/pageObjects/MakePaymentFormPage.js:14]: I thought that is related to the review descr, now I think this is a different part 🤔 good question.
 
> Change from page.waitForTimeout(...) to frame.waitForSelector(...) to ensure Klarna payment page is completely loaded
